.env.dist

```diff
@@ -7,9 +7,16 @@ COMPOSE_PROJECT_NAME=ergonode
 POSTGRES_PASSWORD=123
 POSTGRES_USER=postgres
 EXPOSED_POSTGRES_PORT=5432
-POSTGRES_DB=ergonode
+POSTGRES_DB=postgres
```

Comment:
We use this environment in this time ?

Reply:
in service postgres this is used.

Reply:
but this can be removed , because this values have same values as default PostgreSQL docker image.